samples/python/demo.py

```diff
@@ -49,11 +51,11 @@ def add(self, action):
         self.links[tag] = action
         return "link", tag
 
-    def _enter(self, event):
+    def _enter(self):
         self.text.config(cursor="hand2")
```

Comment:
Why is it changed? (the change is "not compatible")

Reply:
@alalek I found the variable unused after linting the code so i removed it. If that's used anywhere we can add it back.

Reply:
It is a callback function. Lint tool is stupid here.

General rule of fixing something - avoid unnecessary changes.